INSTALL.md

```diff
@@ -24,32 +24,46 @@ If building from git, you'll also need:
 * xmllint
 * xmlto
 
-The commands to install just the first four prerequisites follow.
+The commands to install the prerequisites follow...
 
-On Debian:
+### **Debian**:
```

Comment:
Since you're switching to headings, remove the colon and leave off the bold markup (headings are always bold).

Reply:
Actually it depends on the tool which is rendering the Markdown. E.g. my IDE does not have bold headlines by default. And if it is like that, the readability is much better if using bold.